cl/corpus_importer/tests.py

```diff
@@ -476,6 +478,48 @@ def test_num_queries_ok(self):
             generate_ia_json(3)
 
 
+class TNCorpusTests(TestCase):
+    """Can we properly import the TN Corpus?"""
+
+    fixtures = ["tenn_court_fixture.json"]
+
+    test_dir = os.path.join(
+        settings.INSTALL_ROOT, "cl", "corpus_importer", "test_assets"
+    )
+
+    def tearDown(self):
+        Docket.objects.all().delete()
+
+    def test_import(self):
+        """Can we import two cases successfully"""
+        pre_install_count = OpinionCluster.objects.all().count()
+        filepath = os.path.join(
+            self.test_dir, "tenn_test_files", "tn_corpus_test_asset.json"
+        )
+        file = open(filepath)
```

Comment:
Dangling file handle, again. Sorry if I missed it the first time. I'd suggest you pretend you never learned how to open files this way. The only reason I can think of doing it like this is to prevent indentation that'd be caused by a context handler.

Reply:
Fixed